scripts/build/index.js

```diff
@@ -12,29 +12,59 @@ import write from './write-html'
 
 import rootReducer from '../../src/reducers'
 import { createStore } from 'redux'
-import { getInitialState } from '../../src/getInitialState'
 import { getPopularTags } from '../../src/selectors'
+import {
+  fetchProjectsFromAPI,
```

Comment:
__[eslint]__
* Error - 'fetchProjectsFromAPI' is defined but never used. (no-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->